spec/models/miq_policy_spec.rb

```diff
@@ -207,6 +207,15 @@
       end
     end
 
+    describe ".miq_policies (virtual_has_many)" do
+      before { profiles }
+
+      it "gets the policies under a profile" do
+        expect(MiqPolicySet.find_by(:name => "ps3").miq_policies).to match_array(Array.wrap([policies[0]]))
```

Comment:
I don't under stand the Array.wrap when you are already wrapping it manually in an Array...

```ruby
Array.wrap(["x"]) == ["x"]
```

Reply:
Good point, I'll change that as well as the `describe`. Thanks!